GitCommands/Git/ExecutableExtensions.cs

```diff
@@ -376,7 +376,7 @@ string ComposeOutput()
             var error = outputEncoding.GetString(errorBuffer.GetBuffer(), 0, (int)errorBuffer.Length);
             var exitCode = await process.WaitForExitAsync();
```

Comment:
I think we need to use `await exitTask` there. 
To prevent second run.

Reply:
Done in line 373: `await Task.WhenAll(outputTask, errorTask, exitTask);`.

Reply:
![image](https://user-images.githubusercontent.com/3169265/114910264-57910600-9e26-11eb-8379-7ab6be721507.png)

Reply:
Also.
https://www.meziantou.net/process-waitforexitasync-doesn-t-behave-like-process-waitforexit.htm

Reply:
> I think we need to use `await exitTask` there.
To prevent second run.

Thank you, I got it now. (I think in this case, it will not do harm and will not execute very much twice. Though beware of early signs, i.e. it should be changed.)

The .NET 5 issue does not affect this code because the output tasks are awaited explicitly.

Reply:
This was not changed in this PR, hesitant to change...